symbulate/distributions.py

```diff
@@ -375,25 +375,17 @@ class Normal(Distribution):
 
     Attributes:
       mean (float): mean parameter of the normal distribution
-      var (float): variance parameter of the normal distribution
-      sd (float): standard deviation parameter of the normal 
+      sd (float): standard deviation parameter of the normal
         distribution (if specified, var parameter will be ignored)
+      var (float): variance parameter of the normal distribution
+
     """
     #TODO edit docstring for Normal Distribution
 
-    def __init__(self, mean=0.0, var=1.0, sd=None):
-
+    def __init__(self, mean=0.0, sd=1.0, var=None):
         #Note: cleaner way to implement this
-
-        if sd is None:
-            if var > 0:
-                self.scale = np.sqrt(var)
-            elif var == 0: 
-                raise NotImplementedError
-                #TODO
-            else:
-                raise Exception("var cannot be less than 0")
-        
+        if var is not None:
```

Comment:
I think this is wrong. I should be able to do `Normal(mean=0, var=1)`, but I can't. You need to allow the user to specify `sd=` or `var=`, their choice.

Reply:
Thanks professor. I fixed and tested again. 
![image](https://user-images.githubusercontent.com/23329938/43970015-e509f992-9c80-11e8-9eb6-a32a80b3734d.png)